rest/src/routes/routeUtils.js

```diff
@@ -322,7 +322,7 @@ const routeUtils = {
 	 * @returns {Promise<Uint8Array>} Account public key.
 	 */
 	addressToPublicKey: (db, accountAddress) => db.addressToPublicKey(accountAddress)
```

Comment:
i'm not really happy with this behaviour, is the resulting code clean if this function throws an Error (promise failure), and whoever calls it manages the promise rejection?